index.html

```diff
@@ -9,6 +9,7 @@
 
 <body>
     <p sty>welcome to team-dev</p>
+    <p>welcome to team-dev Hello</p>
```

Comment:
cloud remove `Hello` welcome to team-dev string red

Reply:
sure, can do!